pkg/network/usm/monitor.go

```diff
@@ -321,6 +322,7 @@ func (m *Monitor) GetProtocolStats() map[protocols.ProtocolType]interface{} {
 		// Update update time
 		now := time.Now().Unix()
 		m.lastUpdateTime.Swap(now)
+		telemetry.ReportPrometheus()
```

Comment:
do we need to actively call that? shouldn't it be pulled from an external component?
What will happen if there is no prometheus?